README.md

```diff
@@ -9,7 +9,43 @@ A multi-language library containing implementations of common software design an
 
 ## Articles / Websites
 
+### Model View View-Model (MVVM) - Architectural Pattern
 
+Just like the Model View Controller (MVC) pattern, Model View View-Model (MVVM) is an architectural pattern in software development whose purpose is to provide a clear separation of concerns in applications, especially those with complex user interfaces and interactive features. In both patterns, the program logic of your application (the model) is kept separate from the user interface (the view). The View-Model (and Controller in the case of the MVC pattern) act as the mediator between the Model and View. This separation enhances the structure and maintainability of the code, making it easier to develop, test and manage.  
```

Comment:
- I think the first sentence should have some punctuation to make it easier to read.
- I feel like adding the blurb about the `controller in the case of the MVC pattern` is ok, but could be removed.
- act -> acts
- could add 'security' to the list of benefits

Reply:
Thank you for the suggestions! Changes made.